docs/tutorials/migrations/nvm.md

```diff
@@ -0,0 +1,122 @@
+* **You are already using Flox for your non-JS/non-TS projects.** If you are already using Flox for projects written in other languages, you can streamline your development workflow by using it for your TypeScript and JavaScript projects as well. If you can replace a tool that only manages Node.js versions with something that manages _all_ software dependencies, you're well on your way to simplifying your dev workflow.
+* **You need to manage versions of other JavaScript runtimes, like Bun or Deno.** While nvm is, as the name denotes, used for managing Node.js versions, that's all it does. You can use Flox to install node, but you can also use it to install different JavaScript runtimes, including [Bun](https://bun.sh/){:target="\_blank"} and [Deno](https://deno.com/){:target="\_blank"}.
+
+## Install Flox
+Download and install Flox as described in our [installation instructions][install_flox]{:target="\_blank"}.
+
+## Create a Flox environment in your existing project and install Node.js
+Run `flox init` from the relevant project directory (see [creating environments][creating_tutorial]{:target="\_blank"} for details), and then search for the version of Node.js that is currently included in your `.nvmrc`, or in the `"engines"` property of your `package.json`. (Note that node is listed as `nodejs` and variants thereof in the Flox catalog.)
+
+```zsh
+➜  node-project git:(main) ✗ flox search nodejs
+nodejs       Event-driven I/O framework for the V8 JavaScript engine
+nodejs_14    Event-driven I/O framework for the V8 JavaScript engine
+nodejs_16    Event-driven I/O framework for the V8 JavaScript engine
+nodejs_18    Event-driven I/O framework for the V8 JavaScript engine
+nodejs_19    Event-driven I/O framework for the V8 JavaScript engine
+nodejs_20    Event-driven I/O framework for the V8 JavaScript engine
+nodejs_21    Event-driven I/O framework for the V8 JavaScript engine
+nodejs_22    Event-driven I/O framework for the V8 JavaScript engine
+nodejs_23    Event-driven I/O framework for the V8 JavaScript engine
+nodejs-10_x  Event-driven I/O framework for the V8 JavaScript engine
+
+Showing 10 of 67 results. Use `flox search nodejs --all` to see the full list.
+
+Use 'flox show <package>' to see available versions
+```
+
+There is _a lot_ of software in the Flox catalog, so you should have no trouble finding the Node.js version you need. For this project, perhaps you initially decide to install Node.js v20.18.1, "Iron" (LTS).
+
+```zsh
+➜  node-project git:(main) ✗ flox install nodejs
+✅ 'nodejs' installed to environment 'node-project'
+```
+## Verify the Node.js version
+Now that you've installed Node.js, you activate the Flox environment to verify that it has the the version you expect.
```

Comment:
```suggestion
Now that you've installed Node.js, activate the Flox environment to verify that it has the the version you expect.
```

Reply:
I was going for a historical present usage here to create a sense of narrative, rather than the imperative, which I think changes the tone a bit. I can rework this to see if it's more in tune with what you're expecting here.